### ISSUE TITLE
Json parsing error with threads

### ISSUE THREAD

**Issue by Priyankasahanis** [mrproper/proxmox-ve-api-perl#30]
I am blocked by using this interesting module because it is unable is parse the JSON response. The parent script who is calling the API's are using threads. 

**Sample error:**
2023/08/01 02:31:37 PID5817[undef] WARN TA::Comm::Service::Proxmox::_connect(): 42 : Couldn't log in to proxmox host. ERROR: JSON text must be an object or array (but found number, string, true, false or null, use allow_nonref to allow this) at /usr/local/share/perl/5.26.1/Net/Proxmox/VE/Access.pm line 435.

**Comment by djzort** [comment]
I dont think this is a threads issue? Please try with v0.38 which just was released to CPAN
